### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by otopba**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 0.7.6 (21st June 2020)
+
+- Update image: ^2.1.1 -> 2.1.13
+- Update args: ^1.5.0 -> 1.6.0
+- Update yaml: 2.1.15 -> 2.2.1
+- Format source code
+
```
Comment (MarkOSullivan94) [inline CHANGELOG.md:9]: Could you undo these changes?

I never update the CHANGELOG until right before publishing a new update

Thanks!

Comment (otopba) [reply CHANGELOG.md]: @MarkOSullivan94 Hi! Done

**Comment by MarkOSullivan94** [comment]
Closing as outdated and dependencies have just been updated with #237
